skimage/io/tests/test_mpl_imshow.py

```diff
@@ -78,7 +78,7 @@ def test_low_dynamic_range():
 
 def test_outside_standard_range():
     plt.figure()
-    with expected_warnings(["out of standard range"]):
+    with expected_warnings(["out of standard range|CObject type is marked"]):
```

Comment:
This warning was raised on the Appveyor build, I actually do not know what it means.